docs/Gradle_README.md

```diff
@@ -12,19 +12,26 @@ like (minus all the comments):
 
 ```groovy
 plugins {
-    id 'java'
-    id 'application'
+  id 'java'
+  id 'application'
 }
 
 repositories {
-    jcenter()
+  jcenter()
 }
 
 dependencies {
-    testCompile 'junit:junit:4.12'
+  testImplementation 'org.junit.jupiter:junit-jupiter-api:5.5.2'
+  testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.5.2'
 }
 
-mainClassName = 'Hellos'
+application {
+  mainClassName = 'Hellos'
+}
+
+test {
+  useJUnitPlatform()
+}
 ```
 
 The first two `apply plugin` lines tell Gradle that this is a Java project
```

Comment:
The `apply plugin` wording doesn't seem correct or helpful given the code that I'm looking at. I don't know if that's _always_ been broken, or if the Gradle syntax changed somewhere and the wording didn't.

Reply:
Huh, good catch. We should reword that.